week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -26,8 +34,19 @@ function exerciseOne(arrayOfPeople) {
  */
 function exerciseTwo(shopping) {
   //Write your code in here
+
+  let getContent = document.getElementById("content");
+  let createList = document.createElement("ul");
+  getContent.appendChild(createList)
+  for (let i = 0; i<shopping.length; i++){
```

Comment:
Works perfectly well 👏. Which array method would you use here instead of the for loop?

Reply:
forEach. Why didn't I used it :(( LAzy me